react-sdk/src/hooks/use-suggestions.ts

```diff
@@ -56,6 +56,9 @@ export interface useTamboSuggestionsResultInternal {
     Error,
     AbortController
   >;
+
+  /** The full suggestions query object from React Query */
+  suggestionsQuery: ReturnType<typeof useTamboQuery>;
```

Comment:
so this isn't the *query* but is the query *result* - elsewhere we expose `generateResult` and `acceptResult` - mind making this `suggestionsResult` or something consistent with those names?

For typing, it would be better to expose a `UseTamboQueryResult` from `react-query-hooks.ts` like we do for `UseTamboMutationResult`